Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```
@@ -115,3 +115,37 @@
     capacity: 1
   - type: UseDelay
     delay: 1.5
+
+- type: entity
+  name: taiwan pond
+  parent: [BaseWeaponLauncher, BaseGunWieldable] # A fake taiwan pond for the clown, acts like a normal china lake.
+  id: WeaponLauncherFakeTaiwanPond
+  description: PLOOP.
```

Add a suffix here for the admin spawn menu.
```suggestion
  description: PLOOP.
  suffix: LETHAL
```